src/components/OrganizeCampaignTabs.tsx

```diff
@@ -1,39 +1,81 @@
+    value: string | undefined;
+}
 
 interface OrganizeCampaignTabsProps {
     currentTab?: string;
     onSelectTab: (key : ReactText) => void;
 }
+
+function TabPanel(props:TabPanelProps) {
+    const { children, value, index, ...other } = props;
+    return (
+        <div
+            aria-labelledby={ index }
+            hidden={ value !== index }
+            id={ `tabpanel-${index}` }
+            role="tabpanel"
+            { ...other }>
+            { value === index && (
+                <Box>
+                    { children }
+                </Box>
+            ) }
+        </div>
+    );
+}
+
+const useStyles = makeStyles(() => ({
+    root: {
+        backgroundColor: grey[100],
+        flexGrow: 1,
+    },
+}));
+
 const OrganizeCampaignTabs: FunctionComponent<OrganizeCampaignTabsProps> = ({ children, currentTab, onSelectTab }) => {
     const intl = useIntl();
+    const classes = useStyles();
+    const [value, setValue] = useState(currentTab);
+
+    const handleChange = (event: ChangeEvent<unknown>, newValue: string) => {
+        setValue(newValue);
+        onSelectTab(newValue);
+    };
 
     return (
-        <View flexGrow={ 1 } padding="0 1rem" width="100%">
-            <Tabs aria-label="Campaign Menu"
-                onSelectionChange={ onSelectTab }
-                selectedKey={ currentTab }>
-                <Item key="summary" title={ intl.formatMessage({
+        <div className={ classes.root }>
+            <Tabs aria-label="campaign tabs" indicatorColor="primary" onChange={ handleChange } textColor="primary" value={ value }>
+                <Tab label={ intl.formatMessage({
                     id: 'layout.organize.campaigns.summary',
-                }) }>
-                    <Content>{ children }</Content>
-                </Item>
-                <Item key="calendar" title={ intl.formatMessage({
+                }) } value="summary"
+                />
+                <Tab label={ intl.formatMessage({
                     id: 'layout.organize.campaigns.calendar',
-                }) }>
-                    <View flexGrow={ 1 } maxHeight="90vh">
-                        { children }
-                    </View>
-                </Item>
-                <Item key="insights" title={ intl.formatMessage({
+                }) } value="calendar"
+                />
+                <Tab label={ intl.formatMessage({
                     id: 'layout.organize.campaigns.insights',
-                }) }>
-                    <Content>{ children }</Content>
-                </Item>
+                }) } value="insights"
+                />
             </Tabs>
-        </View>
+            <TabPanel index="summary" value={ value }>
+                { children }
+            </TabPanel>
+            <TabPanel index="calendar" value={ value }>
+                { children }
+            </TabPanel>
+            <TabPanel index="insights" value={ value }>
+                { children }
+            </TabPanel>
```

Comment:
Why three different instances of the same children? I feel like it should be enough to just use a single `TabPanel` for layout (which wouldn't need to be a separate component) without the `index` and `value` props, with the children inside it.